.github/workflows/ci_tests_dev.yaml

```
@@ -81,8 +84,9 @@ jobs:
       - name: Install dependencies
         run: |
           conda install ninja cmake libblas libcblas liblapack fftw gdal \
-                        ghostscript libnetcdf hdf5 zlib curl pcre ipython \
-                        dvc pytest pytest-cov pytest-mpl
+                        ghostscript libnetcdf hdf5 zlib curl pcre make
+          pip install numpy pandas xarray netCDF4 packaging \
```

Just to be extra dangerous, as mentioned at https://github.com/GenericMappingTools/pygmt/pull/1105#pullrequestreview-618117567 :imp:

```suggestion
          pip install --pre numpy pandas xarray netCDF4 packaging \
```